fdbclient/StorageServerInterface.h

```diff
@@ -296,6 +301,9 @@ struct GetKeyValuesRequest : TimedRequest {
 	SpanID spanContext;
 	Arena arena;
 	KeySelectorRef begin, end;
+	// This is a dummy field there has never been used.
```

Comment:
Because of this, I think maybe we should consider sharing the implementation of getRange with different interface rather than an unified interface (the ...MaybeHop...).